sphericalmercator/bq/doc/REFERENCE.md

```diff
@@ -0,0 +1,28 @@
+## Reference
+
+### SPHERICALMERCATOR
+
+#### BBOX
+
+{{% bannerNote type="code" %}}
```

Comment:
Please don't use `NUMERIC`. Also, I have no idea what this is supposed to be returning from the description so that's not a good signal.